src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -95,6 +95,25 @@ public CameraData()
             UpY = defaultCameraUpDirection.Y;
             UpZ = defaultCameraUpDirection.Z;
         }
+        
+        public override bool Equals(object obj)
+        {
+            var other = obj as CameraData;
+            return obj is CameraData && this.Name == other.Name
+                   && Math.Abs(this.EyeX - other.EyeX) < 0.0001
```

Comment:
can you define the .0001 as a variable - like `epsilon` it can be a local variable just to this equal method if you prefer.

Reply:
sure,